### DISCUSSION TITLE
MFA Android

### DISCUSSION THREAD

**Discussion by dacastro4** [timdorr/tesla-api#284] [Q&A]
Hey guys. I've been trying to accomplish the MFA in WearOS but I haven't been able to. Has any you solved this using Android?

**Comment by sdallaire** [comment]
I haven't been able to get past the first query either. Don't know how to run the javascript code from my Android app or to avoid it altogether. Nobody managed to make it work on Android?

**Comment by nrobi144** [comment]
@dacastro4 @sdallaire Here's an Android implementation, supports JP Compose only for now and will be available on `mavenCentral` probably after the holidays https://github.com/octopus-energy/TeslaAndroidAuth
